### ISSUE TITLE
Profanity-Omemo-Plugin in C - there is interest, there is demand...

### ISSUE THREAD

**Issue by macandchief** [reedts/profanity-omemo#2]
First of all thank you for the work on this project.

Just read the thread on the python plugin page and I'd like to assure you that there is demand for a profanity omemo plugin in C as none of the XMPP cli clients have this feature so far.

The python profanity-omemo-plugin looks really good, but it is still work in progress.

I'd like to ask: How does the progress look with this project?

I'm surely not the only one who would love to see an omemo plugin in C as part of Profanity.

**Comment by reedts** [comment]
I'm currently working with @roobre to increase speed of development. Although there were no commits recently that does not mean the project's dead or anything.

We're both ready to continue working on this and hope to make further progress in the near future.

There is still a lot to be done but we should have more time at our hands now :).

**Comment by nadiamoe** [comment]
I agree. It has been some hiatus due to personal circumstances on both of us, but we're getting some free time and advancing on design, testing environments, familiarization with the specs, and code.

Hopefully we'll have some more progress soon!

**Comment by macandchief** [comment]
Great, glad to hear it! Wish you fun with development.
There is nothing like well made CLI tools... And Profanity is still missing a fully working omemo plugin.

**Comment by demonking** [comment]
Would be nice to hear, which are the further steps.
Maybe some people can help, to make profanity more great with this plugin

**Comment by philipflohr** [comment]
Are there any news or plans on this?
